config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,9 @@
+CarrierWave.configure do |config|
+ config.fog_credentials = {
+    provider:              'AWS',         # required
+    aws_access_key_id:     ENV["aws_key"],# required
+    aws_secret_access_key: ENV["aws_secret"],  # required
+    region:                ENV["aws_region"], # optional, defaults to 'us-east-1'
+  }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.